main.go

```diff
@@ -130,17 +132,51 @@ func main() {
 
 	}
 
+	// setup metrics via dropsonse
 	sender := metric_sender.NewMetricSender(dropsonde.AutowiredEmitter())
+	var metricsReporter *metrics.MetricsReporter
+	if c.EnableEnvelopeV1Metrics {
```

Comment:
This still creates the dropsonde emitter regardless of that setting. 

The dropsonde `sender` is also still used in the FD monitor and the NATS monitor. Is that intended?